### PR TITLE
Add precedence rules for sampling

### PR DESCRIPTION
**Pull request by bm1549**
### What does this PR do? What is the motivation?

Clarify precedence rules for sampling since this has been asked by customers

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 1]
```diff
@@ -148,6 +148,22 @@ To specify a specific percentage of a service's traffic to be sent, add an envir
...
+
+1. Remotely configured sampling rules, either set via [Agent](#managing-ingestion-for-all-services-at-the-agent-level) or [resource-based sampling][16]
+1. [Adaptive sampling][17]
+1. [Locally configured sampling rules][8] (`DD_TRACE_SAMPLING_RULES`)
+1. [Remotely configured global sampling rate][8]
+1. [Locally configured global sampling rate][8] (`DD_TRACE_SAMPLE_RATE`)
+1. [Rates from the trace agent controlled indirectly via Agent settings][18] remotely or locally, like `DD_APM_TARGET_TPS` and `DD_APM_MAX_TPS`
+1. In the absence of any sampling rates, the default is 100%, meaning keep all traces
+
+In simpler terms, we follow the following precedence rules
```
Comment (neko-dd) [inline content/en/tracing/trace_pipeline/ingestion_controls.md:162]:
```suggestion
To phrase it another way, Datadog uses the following precedence rules:
```

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 2]
```diff
@@ -148,6 +148,22 @@ To specify a specific percentage of a service's traffic to be sent, add an envir
...
 5. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The ingestion reason for the service is shown as `ingestion_reason:rule`.
 
+### Sampling Precedence Rules
+If sampling rules are set in multiple different locations, the following precedence rules will apply in order of highest to lowest, where higher precedence rules can override lower precedence rules
+
+1. Remotely configured sampling rules, either set via [Agent](#managing-ingestion-for-all-services-at-the-agent-level) or [resource-based sampling][16]
+1. [Adaptive sampling][17]
+1. [Locally configured sampling rules][8] (`DD_TRACE_SAMPLING_RULES`)
+1. [Remotely configured global sampling rate][8]
+1. [Locally configured global sampling rate][8] (`DD_TRACE_SAMPLE_RATE`)
```
[the remark this answers is not in the document]
Comment (bm1549) [reply content/en/tracing/trace_pipeline/ingestion_controls.md:158]: Yes, the section in the other page is fairly overloaded without intermediate sections to break it up

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 3]
```diff
@@ -148,6 +148,22 @@ To specify a specific percentage of a service's traffic to be sent, add an envir
 4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service. **Note**: The service name value is case sensitive. It should match the case of your service name.
 5. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The ingestion reason for the service is shown as `ingestion_reason:rule`.
 
+### Sampling precedence rules
+If sampling rules are set in multiple different locations, the following precedence rule apply in order, where rules that appear first on the list can override lower precedence rules:
```
Comment (bm1549) [inline content/en/tracing/trace_pipeline/ingestion_controls.md:152]:
```suggestion
If sampling rules are set in multiple locations, the following precedence rule apply in order, where rules that appear first on the list can override lower precedence rules:
```

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 4]
```diff
@@ -148,6 +148,22 @@ To specify a specific percentage of a service's traffic to be sent, add an envir
 4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service. **Note**: The service name value is case sensitive. It should match the case of your service name.
 5. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The ingestion reason for the service is shown as `ingestion_reason:rule`.
 
+### Sampling precedence rules
+If sampling rules are set in multiple locations, the following precedence rule apply in order, where rules that appear first on the list can override lower precedence rules:
```
Comment (yuanyuanzhao3) [inline content/en/tracing/trace_pipeline/ingestion_controls.md:152]: nit: plural "rules"?

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 5]
```diff
@@ -148,6 +148,22 @@ To specify a specific percentage of a service's traffic to be sent, add an envir
...
+
+1. Remotely configured sampling rules, either set through the [Agent](#managing-ingestion-for-all-services-at-the-agent-level) or [resource-based sampling][16]
+1. [Adaptive sampling][17]
```
Comment (yuanyuanzhao3) [inline content/en/tracing/trace_pipeline/ingestion_controls.md:155]: nit: "Adaptive sampling configured rules"

File: content/en/tracing/trace_pipeline/ingestion_controls.md [hunk 6]
```diff
@@ -148,6 +148,22 @@ To specify a specific percentage of a service's traffic to be sent, add an envir
 4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service. **Note**: The service name value is case sensitive. It should match the case of your service name.
 5. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The ingestion reason for the service is shown as `ingestion_reason:rule`.
 
+### Sampling precedence rules
+If sampling rules are set in multiple locations, the following precedence rule apply in order, where rules that appear first on the list can override lower precedence rules:
+
+1. Remotely configured sampling rules, either set through the [Agent](#managing-ingestion-for-all-services-at-the-agent-level) or [resource-based sampling][16]
```
Comment (yuanyuanzhao3) [inline content/en/tracing/trace_pipeline/ingestion_controls.md:154]: this should not be here.

**Review by yuanyuanzhao3 — CHANGES_REQUESTED** [review]

File: content/en/tracing/trace_pipeline/ingestion_controls.md (hunk 6 above)
Comment (bm1549) [reply content/en/tracing/trace_pipeline/ingestion_controls.md:154]: Happy to move it - where should it go?

Comment (yuanyuanzhao3) [reply content/en/tracing/trace_pipeline/ingestion_controls.md:154]: I mean remote configured sampling rules are only set through the "Resource-based Sampling" product. The agent managed ingestion does not change that.

Comment (bm1549) [reply content/en/tracing/trace_pipeline/ingestion_controls.md]: Gotcha - just updated

**Review by lucaspimentel — APPROVED** [review]
thanks!

**Review by yuanyuanzhao3 — APPROVED** [review]

**Review by neko-dd — APPROVED** [review]
